src/manifest/owned/pfm.rs

```diff
@@ -408,6 +564,53 @@ mod test {
         );
     }
 
+    #[test]
+    fn parse_firmware_flags() {
```

Comment:
I see what you're doing here, but unfortunately it only means you test round-trips. You aren't testing that specific known-good bit patterns parse to good memory representations, and vice-versa. You'll need to add tests for those, and you can't really "test all of them" this way. =(

The round-trip-tests are, IMO, less useful.

Reply:
resolving, outdated